core-bundle/src/Resources/contao/library/Contao/File.php

```diff
@@ -806,6 +806,19 @@ public function isUnprotected()
 		return (new Folder(\dirname($this->strFile)))->isUnprotected();
 	}
 
+	/**
+	 * Return the full path
+	 */
+	public function getFullPath()
```

Comment:
should we maybe add this as `fullPath` to the `__get` method for a better consistency with the current API of the File class?

Or did you use a method to have the same API as the FilesModel has?

Reply:
I have removed the method again in 19a2f96639ce2eb9bcf5bf091cebee0abbe84728.